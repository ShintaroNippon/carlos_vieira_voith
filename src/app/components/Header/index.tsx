import React from 'react';

export const Header = (): JSX.Element => (
  <header>
    <h1>Voith Weather</h1>
  </header>
);
