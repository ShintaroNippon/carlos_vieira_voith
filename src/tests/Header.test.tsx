import React from 'react';
import { shallow } from 'enzyme';
import { Header } from '../app/components/Header/index';
test('renders the component', () => {
  const component = shallow(<Header />);
  expect(component).toMatchSnapshot();
});

test('renders the component header', () => {
  const wrapper = shallow(<Header />);
  const welcome = <h1>Voith Weather</h1>;
  expect(wrapper.contains(welcome)).toEqual(true);
});
