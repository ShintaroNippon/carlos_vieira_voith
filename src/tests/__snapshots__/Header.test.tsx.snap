// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<header>
  <h1>
    Search for GitHub users
  </h1>
</header>
`;
